workspace/src/control/include/control/control.h

```diff
@@ -0,0 +1,162 @@
+/******************************************************************************
```

Comment:
Do we want to require consistent coding practices across files, i.e. capitalization, function signatures, etc.?